akka-actor-typed/src/main/scala/akka/actor/typed/internal/Supervision.scala

```diff
@@ -80,11 +72,16 @@ private abstract class AbstractSupervisor[I, Thr <: Throwable](strategy: Supervi
     if (strategy.loggingEnabled) {
       val unwrapped = UnstashException.unwrap(t)
       strategy.logLevel match {
-        case Logging.ErrorLevel =>
-          ctx.asScala.log.error(unwrapped, "Supervisor {} saw failure: {}", this, unwrapped.getMessage)
-        case Logging.WarningLevel =>
-          ctx.asScala.log.warning(unwrapped, "Supervisor {} saw failure: {}", this, unwrapped.getMessage)
-        case level => ctx.asScala.log.log(level, "Supervisor {} saw failure: {}", this, unwrapped.getMessage)
+        case Level.ERROR =>
+          ctx.asScala.log.error(s"Supervisor $this saw failure:", unwrapped)
```

Comment:
Wonder if this shouldn't include the exception like:
```
ctx.asScala.log.error(s"Supervisor $this saw failure: $unwrapped", unwrapped)
```

The Throwable parameter is used by logback to print the stack trace, but that is configurable (I believe).